tsconfig.json

```diff
@@ -1,10 +1,14 @@
 {
   "extends": "@tsconfig/node18/tsconfig.json",
   "compilerOptions": {
-    "noEmit": true,
+    "module": "ES2022",
+    "esModuleInterop": true,
     "resolveJsonModule": true,
     "noUncheckedIndexedAccess": true,
   },
+  "ts-node": {
+    "esm": true,
```

Comment:
I frankly don't recall why I needed to use esm here but I remember it fixing a roadblock.

Reply:
@jasonaowen Let me try again with cjs without and see what happens.

Reply:
I think the problems are much deeper than just cjs vs esm. Scripts don't seem to run on main, for example.

Reply:
That sounds like a big problem! Can you say more? Should that be in a separate issue? When did it stop working?

Reply:
I think this was a local issue, either out of date node modules (run `npm ci`) or something like that. It works fine on main now.